common/lib/capa/capa/inputtypes.py

```diff
@@ -1231,6 +1231,7 @@ def get_attributes(cls):
             Attribute('size', '20'),
             Attribute('inline', False),
             Attribute('label', ''),
+            Attribute('trailing_text', ''),
```

Comment:
This attribute should be discussed in the docstring for the input. See the docstring for TextLine.